src/marketing/links/link.scss

```diff
@@ -0,0 +1,47 @@
+.link-mktg {
+  position: relative;
+  display: inline-block;
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:active {
+    outline: none;
+  }
+
+  &::after,
+  &.link-emphasis-mktg::before {
+    position: absolute;
+    bottom: -0.15em;
+    left: 0;
+    width: calc(100% - 1em);
+    height: 2px;
+    pointer-events: none;
+    content: '';
+    // stylelint-disable-next-line primer/colors
+    background-color: currentColor;
+    transition: transform 0.3s ease;
+    transform: scaleX(0);
+    transform-origin: 0 0;
+  }
+
+  &.link-emphasis-mktg::before {
```

Comment:
I'm wondering if you might want to put this animation (and the hover below) within `prefers-reduced-motion` media flag

`@media screen and (prefers-reduced-motion: no-preference) {}`

Reply:
Excellent suggestion! Will do.

I also noticed the uneven padding on the buttons and discussed what is the reason, it seems to have something to do with the custom web font we use in marketing. We have agreed to address it in another PR.